src/components/molecules/FormEditor/FormEditor.tsx

```diff
@@ -134,6 +135,7 @@ const FormEditor = (props: {formSchema: any; formUiSchema?: any}) => {
         const content = mergeManifests(selectedResource.text, formString);
         if (content.trim() !== selectedResource.text.trim()) {
           dispatch(updateResource({resourceId: selectedResource.id, content}));
+          trackEvent(CHANGES_BY_FORM_EDITOR, {resourceKind: selectedResource.kind});
```

Comment:
i think this will generate too much noise - could we track it first when the user moves away from the form editor!? (i.e. selects another tab, resource, etc)